network-store-client/src/main/java/com/powsybl/network/store/client/ConfiguredBusImpl.java

```diff
@@ -0,0 +1,227 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.network.store.client;
+
+import com.powsybl.iidm.network.*;
+import com.powsybl.network.store.model.ConfiguredBusAttributes;
+import com.powsybl.network.store.model.Resource;
+
+import java.util.stream.Stream;
+
+/**
+ * @author Abdelsalem Hedhili <abdelsalem.hedhili at rte-france.com>
+ */
+public class ConfiguredBusImpl extends AbstractIdentifiableImpl<Bus, ConfiguredBusAttributes> implements Bus {
+
+    protected ConfiguredBusImpl(NetworkObjectIndex index, Resource<ConfiguredBusAttributes> resource) {
+        super(index, resource);
+    }
+
+    static ConfiguredBusImpl create(NetworkObjectIndex index, Resource<ConfiguredBusAttributes> resource) {
+        return new ConfiguredBusImpl(index, resource);
+    }
+
+    @Override
+    public String getId() {
+        return resource.getId();
+    }
+
+    @Override
+    public String getName() {
+        return resource.getAttributes().getName();
+    }
+
+    @Override
+    public Network getNetwork() {
+        return index.getNetwork();
+    }
+
+    @Override
+    public VoltageLevel getVoltageLevel() {
+        return index.getVoltageLevel(resource.getAttributes().getVoltageLevelId()).orElse(null);
+    }
+
+    @Override
+    public double getV() {
+        return resource.getAttributes().getV();
+    }
+
+    @Override
+    public Bus setV(double v) {
+        resource.getAttributes().setV(v);
+        return index.getBus(resource.getId()).orElse(null);
+    }
+
+    @Override
+    public double getAngle() {
+        return resource.getAttributes().getAngle();
+    }
+
+    @Override
+    public Bus setAngle(double angle) {
+        resource.getAttributes().setAngle(angle);
+        return index.getBus(resource.getId()).orElse(null);
+    }
+
+    @Override
+    public double getP() {
+        return resource.getAttributes().getP();
+    }
+
+    @Override
+    public double getQ() {
+        return resource.getAttributes().getQ();
+    }
+
+    @Override
+    public Component getConnectedComponent() {
```

Comment:
In case of non yet implemented method, it is preferable to throw an exception, except if this method is called in yur use case and you don't want to block your test.